### ISSUE TITLE
Ticking Entity?

### ISSUE THREAD

**Issue by O13SC3N3** [Laike-Endaril/Dynamic-Stealth#24]
Didn't seem to be an issue before the previous DS updates, lately errors like this keep popping off, Ghostly Mobs, Vi Loot Gers, Primitive Mobs,

https://paste.dimdev.org/tapapitece.mccrash

**Comment by Laike-Endaril** [comment]
The field in question is "careerLevel", which I'm assuming is the trading level of the villager.  I have a feeling about 98% of the blame for this should go to vil-loot-gers, which I see you already made an issue on as well.  I started looking into the issue, but vil-loot-gers does not have a source link, so debugging will have to be done from their end.

I'll leave this open for now though, just in case I come up with an idea, someone posts a source link, etc...

========================

Technical details:
The node related to DS in the stacktrace is an AI filter, which holds an ai task known to incorrectly handle null attack targets (namely EntityAIZombieAttack).  That being said, if the filter had activated, the stacktrace would've ended on that node, so in this case the attack target was not null, and the filter allowed the ai task to execute.

I'm pretty sure this has nothing to do with the error though.
Edit: I'll link this issue in their own issue in case they find the information I posted useful
